src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss

```diff
@@ -1,3 +1,7 @@
+.container {
+  padding: 0;
+}
```

Comment:
let's not rewrite the bootstrap class, but use p-0 class instead 
https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
done